src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Compile.cs

```diff
@@ -176,5 +176,41 @@ public void FractionDigitsMismatch_Throws()
             Assert.Contains("fractionDigits", ex.Message);
             Assert.DoesNotContain("totalDigits", ex.Message);
         }
+
+        [Fact]
+        public void TotalDigitsParseValue_Succeeds()
+        {
+            string schema = @"<?xml version='1.0' encoding='utf-8' ?>
+<xs:schema elementFormDefault='qualified'
+           xmlns:xs='http://www.w3.org/2001/XMLSchema'>
+    <xs:simpleType name='foo'>
+        <xs:restriction base='xs:decimal'>
+            <xs:totalDigits value='8' fixed='true'/>
+        </xs:restriction>
+    </xs:simpleType>
+    <xs:simpleType name='bar'>
+        <xs:restriction base='foo'>
+            <xs:totalDigits value='8'/>
+        </xs:restriction>
+    </xs:simpleType>
+</xs:schema>
+";
+            XmlSchemaSet ss = new XmlSchemaSet();
+            ss.Add(null, XmlReader.Create(new StringReader(schema)));
+
+            bool success;
+            try
+            {
+                ss.Compile();
+                success = true;
+            }
+            catch (Exception ex)
```

Comment:
Please remove try catch, we would like to see the exception thrown in case of failure, just:
```csharp
            XmlSchemaSet ss = new XmlSchemaSet();
            ss.Add(null, XmlReader.Create(new StringReader(schema)));
            ss.Compile();
            Assert.True(true);
```
would be enough to check its compiled successfully

Reply:
@buyaa-n I think you meant to write something else instead of always passing assert in the last line of the snippet

Reply:
@buyaa-n, @krwq 
I've updated the test per the above (removed suppression of Exception).  I believe the always passing Assert is reasonable as we're simply looking for ```ss.Compile();``` to not throw an InvalidCastException.  So I've kept it like that.

Reply:
> @buyaa-n I think you meant to write something else instead of always passing assert in the last line of the snippet

@krwq  what else you think is better to write/check after compile, please suggest, as @mrj001 mentioned my suggestion was to confirm that it compiled successfully